### PR TITLE
Standarized the Python dependencies section for cloud and docker modules

### LINKED ISSUE #6081: Refactor external integrations dependencies sections
In #5994, the minimum necessary Python version required in the `Dependencies` section for the Wazuh agents was bumped to version `3.7`. Python `3.6` Security Support has ended since December 2021, so the team has decided to drop compatibility with it and update to the next minor version (`3.7`).

The mentioned modification was not made for the Azure, GCP, and Docker Listener modules, which led the team to discuss the current approach to the Python installation for the agents and the related pip dependencies.
 
We decided that the section for the different modules (AWS, Azure, GCP, and Docker Listener) need the following modifications:
- Centralize the Python reference on a single page, avoiding the problem which led to this issue. 
- Remove the Python installation steps and only mention the range of compatible versions (`3.7+`). 
- Since the dependencies required for the modules must work with any Python stable version, we will only indicate the direct packages needed for each module to work, transferring the responsibility for the indirect dependencies to `pip` which will install the necessary versions.

### PR DESCRIPTION
**Pull request by EduLeon12**
|Related issue|
|---|
|Closes #6081 |

## Description

As explained in #6081, Having a Python dependencies section for each cloud and docker-listener module caused an issue because upgrading python version and not updating the documentation in each section caused a dependency issue.

The agent section was changed to have only one Python dependencies section and modules were changed to reference to it. 

## Checks
- [x] Compiles without warnings.
- [x] Uses present tense, active voice, and semi-formal registry.
- [x] Uses short, simple sentences.
- [x] Uses **bold** for user interface elements, _italics_ for key terms or emphasis, and `code` font for Bash commands, file names, REST paths, and code.
- [x] Uses three spaces indentation.
- [x] Adds or updates meta descriptions accordingly.
- [x] Updates the `redirects.js` script if necessary (check [this guide](https://github.com/wazuh/wazuh-documentation/blob/master/NEW_RELEASE.md)).

### CODE REVIEW

File: source/gcp/prerequisites/dependencies.rst [hunk 1]
```diff
@@ -7,63 +8,15 @@
...
------------------------------
-
-`google-cloud-pubsub <https://pypi.org/project/google-cloud-pubsub/>`_ is the official python library supported by Google to manage Google Cloud Pub/Sub resources. It is used to pull the log messages from the Pub/Sub queue.
-
-To install the dependencies, execute the following command:
-
-  .. code-block:: console
-
-    # pip3 install cachetools==4.1.0 certifi==2022.12.07 cffi==1.14.4 chardet==3.0.4 charset-normalizer==2.0.4 google-api-core==1.30.0 google-auth==1.28.0 google-cloud-core==1.7.1 google-cloud-pubsub==2.7.1 google-cloud-storage==1.39.0 google-crc32c==1.1.2 google-resumable-media==1.3.1 googleapis-common-protos==1.51.0 grpc-google-iam-v1==0.12.3 grpcio==1.38.1 idna==2.9 libcst==0.3.20 mypy-extensions==0.4.3 packaging==20.9 proto-plus==1.19.0 protobuf==3.19.6 pyasn1-modules==0.2.8 pyasn1==0.4.8 pycparser==2.20 pyparsing==2.4.7 pytz==2020.1 PyYAML==5.4.1 requests==2.25.1 rsa==4.7.2 setuptools==59.6.0 six==1.14.0 typing-extensions==3.10.0.2 typing-inspect==0.7.1 urllib3==1.26.5
+  # pip3 install google-cloud-core==1.7.1 google-cloud-pubsub==2.7.1 google-cloud-storage==1.39.0 cffi==1.14.4 chardet==3.0.4  pyparsing==2.4.7 pytz==2020.1
```
Comment (nico-stefani) [inline source/gcp/prerequisites/dependencies.rst:22]: The direct dependencies here are `google-cloud-storage==1.39.0 google-cloud-pubsub==2.7.1`, the others are indirect

```
google-cloud-pubsub==2.7.1
  - google-api-core [required: >=1.26.0,<3.0.0dev, installed: 1.30.0]
    - google-auth [required: >=1.25.0,<2.0dev, installed: 1.28.0]
      - cachetools [required: >=2.0.0,<5.0, installed: 4.1.0]
      - pyasn1-modules [required: >=0.2.1, installed: 0.2.8]
        - pyasn1 [required: >=0.4.6,<0.5.0, installed: 0.4.8]
      - rsa [required: >=3.1.4,<5, installed: 4.7.2]
        - pyasn1 [required: >=0.1.3, installed: 0.4.8]
      - setuptools [required: >=40.3.0, installed: 59.6.0]
      - six [required: >=1.9.0, installed: 1.14.0]
    - googleapis-common-protos [required: >=1.6.0,<2.0dev, installed: 1.51.0]
      - protobuf [required: >=3.6.0, installed: 3.19.6]
    - packaging [required: >=14.3, installed: 20.9]
      - pyparsing [required: >=2.0.2, installed: 2.4.7] *
    - protobuf [required: >=3.12.0, installed: 3.19.6]
    - pytz [required: Any, installed: 2020.1] *
    - requests [required: >=2.18.0,<3.0.0dev, installed: 2.25.1]
      - certifi [required: >=2017.4.17, installed: 2022.12.7]
      - chardet [required: >=3.0.2,<5, installed: 3.0.4] *
      - idna [required: >=2.5,<3, installed: 2.9]
      - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.5]
    - setuptools [required: >=40.3.0, installed: 59.6.0]
    - six [required: >=1.13.0, installed: 1.14.0]
  - grpc-google-iam-v1 [required: >=0.12.3,<0.13dev, installed: 0.12.3]
    - googleapis-common-protos [required: >=1.5.2,<2.0.0dev, installed: 1.51.0]
      - protobuf [required: >=3.6.0, installed: 3.19.6]
    - grpcio [required: >=1.0.0,<2.0.0dev, installed: 1.38.1]
      - six [required: >=1.5.2, installed: 1.14.0]
  - grpcio [required: >=1.38.1,<2.0dev, installed: 1.38.1]
    - six [required: >=1.5.2, installed: 1.14.0]
  - libcst [required: >=0.3.10, installed: 0.3.20]
    - pyyaml [required: >=5.2, installed: 5.4.1]
    - typing-extensions [required: >=3.7.4.2, installed: 3.10.0.2]
    - typing-inspect [required: >=0.4.0, installed: 0.7.1]
      - mypy-extensions [required: >=0.3.0, installed: 0.4.3]
      - typing-extensions [required: >=3.7.4, installed: 3.10.0.2]
  - packaging [required: >=14.3, installed: 20.9]
    - pyparsing [required: >=2.0.2, installed: 2.4.7]
  - proto-plus [required: >=1.7.1, installed: 1.19.0]
    - protobuf [required: >=3.12.0, installed: 3.19.6]

google-cloud-storage==1.39.0
  - google-auth [required: >=1.11.0,<2.0dev, installed: 1.28.0]
    - cachetools [required: >=2.0.0,<5.0, installed: 4.1.0]
    - pyasn1-modules [required: >=0.2.1, installed: 0.2.8]
      - pyasn1 [required: >=0.4.6,<0.5.0, installed: 0.4.8]
    - rsa [required: >=3.1.4,<5, installed: 4.7.2]
      - pyasn1 [required: >=0.1.3, installed: 0.4.8]
    - setuptools [required: >=40.3.0, installed: 59.6.0]
    - six [required: >=1.9.0, installed: 1.14.0]
  - google-cloud-core [required: >=1.4.1,<2.0dev, installed: 1.7.1] *
    - google-api-core [required: >=1.21.0,<2.0.0dev, installed: 1.30.0]
      - google-auth [required: >=1.25.0,<2.0dev, installed: 1.28.0]
        - cachetools [required: >=2.0.0,<5.0, installed: 4.1.0]
        - pyasn1-modules [required: >=0.2.1, installed: 0.2.8]
          - pyasn1 [required: >=0.4.6,<0.5.0, installed: 0.4.8]
        - rsa [required: >=3.1.4,<5, installed: 4.7.2]
          - pyasn1 [required: >=0.1.3, installed: 0.4.8]
        - setuptools [required: >=40.3.0, installed: 59.6.0]
        - six [required: >=1.9.0, installed: 1.14.0]
      - googleapis-common-protos [required: >=1.6.0,<2.0dev, installed: 1.51.0]
        - protobuf [required: >=3.6.0, installed: 3.19.6]
      - packaging [required: >=14.3, installed: 20.9]
        - pyparsing [required: >=2.0.2, installed: 2.4.7] *
      - protobuf [required: >=3.12.0, installed: 3.19.6]
      - pytz [required: Any, installed: 2020.1] *
      - requests [required: >=2.18.0,<3.0.0dev, installed: 2.25.1]
        - certifi [required: >=2017.4.17, installed: 2022.12.7]
        - chardet [required: >=3.0.2,<5, installed: 3.0.4]
        - idna [required: >=2.5,<3, installed: 2.9]
        - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.5]
      - setuptools [required: >=40.3.0, installed: 59.6.0]
      - six [required: >=1.13.0, installed: 1.14.0]
    - google-auth [required: >=1.24.0,<2.0dev, installed: 1.28.0]
      - cachetools [required: >=2.0.0,<5.0, installed: 4.1.0]
      - pyasn1-modules [required: >=0.2.1, installed: 0.2.8]
        - pyasn1 [required: >=0.4.6,<0.5.0, installed: 0.4.8]
      - rsa [required: >=3.1.4,<5, installed: 4.7.2]
        - pyasn1 [required: >=0.1.3, installed: 0.4.8]
      - setuptools [required: >=40.3.0, installed: 59.6.0]
      - six [required: >=1.9.0, installed: 1.14.0]
    - six [required: >=1.12.0, installed: 1.14.0]
  - google-resumable-media [required: >=1.3.0,<2.0dev, installed: 1.3.1]
    - google-crc32c [required: >=1.0,<2.0dev, installed: 1.1.2]
      - cffi [required: >=1.0.0, installed: 1.14.4] *
        - pycparser [required: Any, installed: 2.20]
    - six [required: >=1.4.0, installed: 1.14.0]
  - requests [required: >=2.18.0,<3.0.0dev, installed: 2.25.1]
    - certifi [required: >=2017.4.17, installed: 2022.12.7]
    - chardet [required: >=3.0.2,<5, installed: 3.0.4]
    - idna [required: >=2.5,<3, installed: 2.9]
    - urllib3 [required: >=1.21.1,<1.27, installed: 1.26.5]
```

**Review by nico-stefani — CHANGES_REQUESTED** [review]

File: source/gcp/prerequisites/dependencies.rst (hunk 1 above)
Comment (EduLeon12) [reply source/gcp/prerequisites/dependencies.rst:22]: After testing the proposed changes the conclusion is:

Command will be:

```
pip3 install google-cloud-core==1.7.1 google-cloud-pubsub==2.7.1 google-cloud-storage==1.39.0 pytz==2020.1
```

**Review by nico-stefani — APPROVED** [review]
LGTM!

File: source/user-manual/agents/cloud-and-container-monitoring.rst [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+.. _cloud_and_container_monitoring:
+
+Cloud and container monitoring
+==============================
+
+Wazuh offers the ability to monitor your cloud and container infrastructure, helping you maintain the highest levels of
+security. In this section, we will show how to configure your Wazuh agent to monitor various cloud providers and container technologies.
+
+To learn more about cloud security, check out :doc:`this section </monitoring>`.
```
Comment (s-ocando) [inline source/user-manual/agents/cloud-and-container-monitoring.rst:14]:
```suggestion
See the :doc:`Cloud security </monitoring>` and :doc:`Container security </container-security/index>` sections to learn more.
```

**Review by s-ocando — COMMENTED** [review]
Hi @EduLeon12, 

Please review my suggestions below.

**Comment by EduLeon12** [comment]
Hi @s-ocando, 

Thanks for the reply and the suggested changes, they have been addressed.

I will wait for your review. 

Regards,

**Pull request closed without merging**